lib/keeper/controller.rb

```diff
@@ -7,22 +7,36 @@ def self.included(klass)
     end
 
     module InstanceMethods
+      JWT_REGEX = /[A-Za-z0-9\-_=]+\.[A-Za-z0-9\-_=]+\.?[A-Za-z0-9\-_=]+/
```

Comment:
i don't think regex matching is necessary, and is off spec. We can just run the jwt though the decoder if it doesn't decode it's def invalid